app/models/ems_refresh.rb

```diff
@@ -140,7 +140,7 @@ def self.get_target_objects(target, single_id = nil)
       ids.uniq!
 
       recs = if target_class <= ManagerRefresh::Target
-               ids.map { |x| ManagerRefresh::Target.load(x) }
+               ids.map { |x| ManagerRefresh::Target.load(x.symbolize_keys) }
```

Comment:
Why is this needed?  Curious if the ManagerRefresh::Targer.load clas can just accept anything and *it* can symbolize the keys.  Also, should this be a `deep_symbolize_keys`?

Reply:
yeah, we can probably move that down